.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,8 +11,11 @@ to conform to the rules below are likely to be rejected.**
     go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
     Create a subfolder with your name, and place the mods in a subfolder with
     a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
-  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
-    do not contain any spaces.
+  * [ ] Folder and file naming:
+    * Folders and filenames shouldn't contain spaces. Only letters `a-z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
+    * Primary color: `part_xyz.stl`
+    * Accent color: `[a]_part_xyz.stl`
```

Comment:
@newphreak are we going to add alternates like `[o]` opaque, `[t]` transparent, etc given the stealthburner and more LED modes in the future?

Reply:
* Opaque color (Blocks light): `[o]_part_xyz.stl`
* Transparent color (Allows light): `[t]_part_xyz.stl`

Reply:
smart, i forgot about those.

Reply:
Doesn't stealthburner uses `[c]` for clear, rather than transparent.